modules/doc/content/help/faq/conda_issues.md

```diff
@@ -104,14 +104,20 @@ Conda issues can be the root cause for just about any issue on this page. Scroll
 
   +Hint:+ For Linux, this will most likely be `conda init bash`. For Macintosh, this can either be bash or zsh.
 
+  !alert note title=mamba init
+  When using `mambaforge3` you might also need to perform a `mamba init` in order to properly use the
+  `mamba` command to install packages. We recommend this as an alternative to conda, as mamba can take
+  advantage of multiple cores to perform tasks. This means it will generally be faster at performing
+  installations than standard conda. Otherwise, they are interchangeable.
+
 - #### Your issue not listed
 
   The quick fix-attempt, is to re-install the moose-packages:
 
   ```bash
   conda deactivate
-  conda remove moose --all --yes
-  conda create moose moose-libmesh moose-tools
+  conda env remove -n moose
+  conda create -n moose moose-libmesh moose-tools
```

Comment:
I did not end up doing this on my laptop this morning, and it seems to still work (so many fingers crossed). For what it's worth, I ended up using 
```
mamba env remove -n moose
mamba create --name moose -q -y
mamba activate moose
mamba install moose-tools moose-libmesh
```
as given on this original page: https://mooseframework.inl.gov/getting_started/installation/conda.html

Reply:
`conda create -n moose moose-libmesh moose-tools` is a shorthand to create an environment and install packages without activating. So it combines lines 2 and 4 in your example essentially. Should be the same. I can make these more explicitly identical if you'd prefer.

Reply:
nope. fewer lines to copy means fewer lines to mess up. Just double checking that we were all on the same page :-)

Reply:
although, do you want to use `mamba` instead of `conda` in the commands, so people can use the copy and paste functionality of the text boxes with the faster option?

Reply:
I think I'm going to use the shorthand on the getting started instructions too - you are right that fewer lines to copy means fewer opportunities to screw up.

Reply:
we should try to use `mamba` where we can... I think at this time, the only subcommand not supported by mamba, is when you supply `config`. Which is necessary to add channels:

```bash
conda config --add channels https://conda.software.inl.gov/public
```
as opposed to (the following fails):
```bash
$> mamba config --add channels https://conda.software.inl.gov/public
<trimmed>
Currently, only install, create, list, search, run, info, clean, activate and deactivate are supported through mamba.
```